src/components/Sidebar/Sidebar.jsx

```diff
@@ -294,7 +294,10 @@ const Sidebar = createClass({
 						{barProps.children}
 					</div>
 				</BarPane>
-				<SplitVertical.Divider className={cx('&-Divider')}>
+				<SplitVertical.Divider
+					className={cx('&-Divider')}
+					style={{ width: '7px' }}
```

Comment:
`style={{ width: 7 }}`

also, why not set this in Sidebar.less?

Reply:
because it's in inline style in `SplitVertical.Divider`

Reply:
fixed